mapbox-gl/mapbox-gl-tests.ts

```diff
@@ -0,0 +1,445 @@
+// These examples are from Mapbox (https://www.mapbox.com/mapbox-gl-js/examples/) and therefore under the copyright of Mapbox Inc.
```

Comment:
The test file was originally written by someone else. I went through and fixed them up a bit as well as adding some additional parts and removing some potentially unneeded ones. Hopefully that might avoid copyright issues (especially since it was just pieces of the code in the first place).